tests/functional/test_package.py

```diff
@@ -732,8 +732,10 @@ def test_can_create_app_packager_with_no_autogen(tmpdir):
     appdir = _create_app_structure(tmpdir)
```

Comment:
From what I could tell it doesn't look like this test `test_can_create_app_packager_with_no_autogen` has anything to do with the `deploy --no-autogen-policy` command so I changed it to create the policy automatically so as not to trigger the `RuntimeError`. If it does let me know and I can make whatever changes are needed.